docs/source/examples/get_label_statistics.rst

```diff
@@ -0,0 +1,67 @@
+######################
+ Get Label Statistics
+######################
+
+This topic describes the get label statistics operation.
+
+Label statistics of dataset could be obtained via :func:`~tensorbay.client.dataset.DatasetClientBase.get_label_statistics`
+as follows:
+
+    >>> from tensorbay import GAS
+    >>> ACCESS_KEY = "Accesskey-*****"
+    >>> gas = GAS(ACCESS_KEY)
+    >>> dataset_client = gas.get_dataset("targetDataset")
+    >>> statistics = dataset_client.get_label_statistics()
+    >>> statistics
+    Statistics {
+        'BOX2D': {...},
+        'BOX3D': {...},
+        'KEYPOINTS2D': {...}
+    }
+ 
+The details of the statistics for the targetDataset are as follows:
```

Comment:
If the statistics have exact numbers, should we use an open dataset as the example?